docs/pages/application-access/getting-started.mdx

```diff
@@ -178,11 +178,22 @@ Make sure to update `apps[0].name` and `apps[0].uri` accordingly if you're using
 
 ### Create a user
 
-Next, let's create a user to access the application we've just connected. Teleport has a built-in role called `access` that allows users to access cluster resources. Create a local user assigned this role:
+<ScopedBlock scope={["oss"]}>
+Next, let's create a user to access the application we've just connected. Teleport has a built-in role called `access` that allows users to access cluster resourcesCreate a local user assigned to these roles:
 
 ```code
 $ tctl users add --roles=access alice
 ```
+</ScopedBlock>
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
+Next, let's create a user to access the application we've just connected. Teleport has a built-in role called `access` that allows users to access cluster resources,
+along with a built-in role called `requester` that allows users to request resources. Create a local user assigned to these roles:
```

Comment:
if I read "`access` that allows users to access cluster resources" + "`requester` that allows users to request resources" as a normal teleport user I'm going to be really confused (actually, I am a bit confused myself). If the `requester` role is relevant to this guide can we say why it's actually needed?

Reply:
I think I'll remove it from this guide, might makes things simpler.